src/routes/index.ts

```diff
@@ -158,7 +158,7 @@ const defaultRouteHandler: RequestHandler<{}, {}, z.infer<typeof DefaultRouteReq
     await handler(req, res, next);
   } else {
     logger.log('error', 'no step');
-    next(createError(500));
```

Comment:
This probably should stay as 500 as this is the catch all around the main application logic and isn't necessarily about bad user request any more.

Reply:
Done!